libmscore/rendermidi.cpp

```diff
@@ -2303,9 +2303,7 @@ void Score::createGraceNotesPlayEvents(const Fraction& tick, Chord* chord, int&
                   el.append(nel);
                   }
 
-            if (gc->playEventType() == PlayEventType::InvalidUser)
-                  gc->score()->undo(new ChangeEventList(gc, el));
-            else if (gc->playEventType() == PlayEventType::Auto) {
+            if (gc->playEventType() == PlayEventType::Auto) {
                   for (int ii = 0; ii < int(nn); ++ii)
                         gc->notes()[ii]->setPlayEvents(el[ii]);
```

Comment:
Since we already have `Chord::setNoteEventLists()` and are modifying this code anyway, could perhaps this `for` loop be replaced with a call to that function?

Reply:
Great catch!